ranger/core/fm.py

```diff
@@ -245,12 +245,16 @@ def _get_image_displayer(self):  # pylint: disable=too-many-return-statements
         return ImageDisplayer()
 
     def _get_thisfile(self):
+        if not self.thistab:
```

Comment:
I'm not sure these should fail silently. Is this necessary?

Reply:
Well, right now the check is also being made in the corresponding macro lambdas so I can remove those. However, if some part of the application calls any of those function at startup time, before initialization, a error will be raised "NoneType does not have attribute thisfile". But you are probalby right. If this is happening something is most likely gone wrong or ranger is being used wrong.